go.mod

```diff
@@ -30,7 +30,11 @@ require (
 	github.com/pingcap/goleveldb v0.0.0-20191226122134-f82aafb29989
 	github.com/pingcap/kvproto v0.0.0-20200518112156-d4aeb467de29
 	github.com/pingcap/log v0.0.0-20200511115504-543df19646ad
+<<<<<<< HEAD
```

Comment:
@bobotu 
Please fix the conflict.

Reply:
Waiting for PR in pingcap/parser to be merged.